python/ray/rllib/pg/pg_evaluator.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+from ray.rllib.models import ModelCatalog
+from ray.rllib.optimizers import Evaluator
+from ray.rllib.pg.policy import PGPolicy
+from ray.rllib.utils.filter import NoFilter
+from ray.rllib.utils.process_rollout import process_rollout
+from ray.rllib.utils.sampler import SyncSampler
+
+# Evaluator for vanilla policy gradient
+
+
+class PGEvaluator(Evaluator):
+    """Evaluator for simple policy gradient."""
+
+    def __init__(self, registry, env_creator, config):
+        self.env = ModelCatalog.get_preprocessor_as_wrapper(
+                registry, env_creator(config["env_config"]), config["model"])
+        self.config = config
+        self.registry = registry
```

Comment:
we don't need this right?

Reply:
I think we still need self.config for the discount factor.

Reply:
ah this comment was in reference for `self.registry` but has since been removed.